app/controllers/Application.java

```diff
@@ -1477,12 +1508,40 @@ public static Result restFlowMetricsGraphData(String flowDefId) {
         .order()
         .desc(AppResult.TABLE.FINISH_TIME)
         .setMaxRows(JOB_HISTORY_LIMIT)
-        // The 2nd and 3rd table are not required for plotting the graph
-        //.fetch(AppResult.TABLE.APP_HEURISTIC_RESULTS, AppHeuristicResult.getSearchFields())
-        //.fetch(AppResult.TABLE.APP_HEURISTIC_RESULTS + "."
-        //    + AppHeuristicResult.TABLE.APP_HEURISTIC_RESULT_DETAILS, "*")
         .findList();
 
     return results;
   }
+
+  private static class AppResourceUsageData {
```

Comment:
I'm not sure if this should be a static class. We're creating instance of this class inside getUserResourceUsage method and we want to have separate instances for each invocation of  getUserResourceUsage. Keeping this static might result in incorrect results if getUserResourceUsage is called concurrently.